### ISSUE TITLE
Missing values in features

### ISSUE THREAD

**Issue by ablaom** [IQVIA-ML/LightGBM.jl#93]
Do LightGBM.jl models allow features to have `missing` values? If so, it would be good to allow this through the MLJ interface as well (probably just need to extend `input_scitype` for the models). I would be happy to make the necessary PR.

@yalwan-iqvia

**Comment by yalwan-iqvia** [comment]
Yes but we have to deal with transposing the "missings" to NaN (because thats how its encoded to the library). So, right now, we can't just blanket turn it on -- I am operating under the assumption here that `missing` means the native julia Missing type and not "encoded missing". 

Also transposing to NaN comes with its troubles like making the translation without paying high memory usage costs, and potential loss of precision when moving from Int -> Float. 

I'm not saying no, I'm saying there is more to consider off the bat than just extending the scitype.

**Comment by ablaom** [comment]
> I am operating under the assumption here that missing means the native julia Missing type 

Yes, I expect that's the main use-case.

Thanks for considering this. Appreciate there is devil in the details and this could take some time.

**Comment by yaxxie** [comment]
@citron could you elaborate on your use case? This ticket is concerned with LightGBM.jl "announcing" support for missing values to MLJ users -- but if you're talking about CLI or python it would seem to me you wish to use LightGBM.jl instead.
